worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/PartitionFilesSorter.java

```diff
@@ -564,7 +564,7 @@ class FileSorter {
       this.originFilePath = fileInfo.getFilePath();
       this.sortedFilePath = Utils.getSortedFilePath(originFilePath);
       this.isHdfs = fileInfo.isHdfs();
-      this.isWarmUp = !isHdfs && reservedMemoryEnabled;
+      this.isWarmUp = !isHdfs && prefetchEnabled;
```

Comment:
can we rename isWarmUp to prefetch?

Reply:
anyway, this is an internal flag, we can change it anytime without bothering users